aiidalab/app.py

```diff
@@ -539,16 +673,16 @@ def _check_modified_state(self):
         if blocked:
             raise RuntimeError("Unable to perform operation, there are local modifications to the app repository.")
 
-    def _install_version(self, _):
+    def _install_version(self, _=None):
```

Comment:
I guess there are times when this will be called without supplying a non-used parameter then? Will it ever be called _with_ a non-used parameter?

Reply:
Yes, it is sometimes called with an argument, however, we do not use that argument hence why it is named `_`.

Reply:
Sure. I just didn't think this function would be called both with and without a parameter.
Is there a way to avoid this or does it not really matter? I'm more thinking about the "why" of this. But maybe not so important for now...

Reply:
This kind of pattern actually appears in various places whenever we use a function stand-alone (without the argument), and register as a callback function with an ipywidget, that will call it with itself as first argument. I'm not sure if there is a better way to do this.

Reply:
Yeah, it's those pescy ipywidgets callbacks. Just didn't find the place this function was used for it. I don't think there is another way. Just curious about the double-effect and why it wouldn't be used for just one or the other.